apps/airnow/airnow.star

```diff
@@ -0,0 +1,157 @@
+"""
+Applet: AirNowAQI
+Summary: Air Now AQI
+Description: Displays the current AQI value and level by location using data provided by AirNow.gov.
+Author: mjc-gh
+"""
+
+load("cache.star", "cache")
```

Comment:
We recently added cache support to the HTTP module. Could simplify your code a bit, and also future proof against potential upcoming changes to `cache.star`.